lib/matplotlib/axes/_axes.py

```diff
@@ -2705,11 +2705,11 @@ def errorbar(self, x, y, yerr=None, xerr=None,
 
         Parameters
         ----------
-        x : scalar
-        y : scalar
+        x : scalar or array-like
```

Comment:
I think that it should be removed for `x` and `y` but retained for `errx` and `erry`. `x` and `y` are arrays to plot. As @anntzer says, scalars are just a special case that represents a dataset of size 1. For `errx` and `erry`, scalars have a somewhat special meaning, which should be emphasized in the docs. Will update.